README.md

```diff
@@ -44,11 +44,14 @@ The object has the following schema (validated [here](./lib/index.js) using [Joi
   * `done: (err: Error, key: String)` - A function that takes an error as the first parameter and the bucket key as the second parameter.
 
 **Optional**
+* `enabled: Boolean default(true)` - if true rate limiting will be enabled for all routes. Setting this `false` will only rate limit routes which have the plugin defined
```

Comment:
I've been thinking a bit more and we can probably change this to `allRoutes` as you suggested.
if it's set to `true` a user can still disable a specific route by setting the route settings to `false` instead of an object
```js
config: {
    plugins: {
      patova: false
    }
}
```
So we can accommodate both use cases this way.